### PR TITLE
Update geom in community district schema and create migration file

### PR DESCRIPTION
**Pull request by pratishta**
Closes https://github.com/NYCPlanning/ae-zoning-api/issues/334

### CODE REVIEW

File: db/migration/0013_previous_lenny_balinger.sql
Comment (pratishta) [inline db/migration/0013_previous_lenny_balinger.sql:1]: For "The migration drops any existing data in the liFt column because they are multipoints" is this something we'd manually add the SQL for here or is there a way to do it with Drizzle?

Comment (TangoYankee) [reply db/migration/0013_previous_lenny_balinger.sql:1]: I'll run this locally tomorrow. If it starts shouting errors, we'll have some answers

Comment (TangoYankee) [reply db/migration/0013_previous_lenny_balinger.sql:1]: Yes, it shouts errors- specifically about multipoints not being multipolygons. I'll share a code snippet for that.

Comment (TangoYankee) [inline db/migration/0013_previous_lenny_balinger.sql:1]:
```suggestion
UPDATE "community_district" SET "li_ft" = NULL; --> statement-breakpoint
ALTER TABLE "community_district" ALTER COLUMN "li_ft" SET DATA TYPE geometry(multiPolygon,2263);
```

Comment (pratishta) [reply db/migration/0013_previous_lenny_balinger.sql:2]: Updated. To confirm, when we want to do things like drop columns, we should manually edit the migration files?

Comment (TangoYankee) [reply db/migration/0013_previous_lenny_balinger.sql:2]: I find it's a grey area on whether it's better to create [a custom migration file](custom) from scratch or edit an auto-generated one. My guideline is that it's okay to edit the auto-generated migration as long as it doesn't contradict the snapshot meta-data.

It's important to highlight that we never [`DROP`](https://www.w3schools.com/SQL/sql_ref_drop_column.asp) the "li_ft" column. That would push us into "temporarily contradicting" the snapshot meta-data. We only update the content of the rows to remove any errant data that is incompatible with the incoming data structure.

**Review by TangoYankee — COMMENTED** [review]
Holding off on approving until after demo at 2 (lest we realize a mistake too late)

**Review by TangoYankee — APPROVED** [review]

File: db/migration/0013_previous_lenny_balinger.sql
Comment (TylerMatteo) [reply db/migration/0013_previous_lenny_balinger.sql:2]: I strongly recommend against manually editing generated migrations. That's generally a big no-no for ORMs with migration generators and I don't see why Drizzle would be any different.

Comment (horatiorosa) [reply db/migration/0013_previous_lenny_balinger.sql:2]: Yeah, we should  only fix a migration with another migration.

Comment (TangoYankee) [reply db/migration/0013_previous_lenny_balinger.sql:2]: Can you expand on why it's a "no-no"?

Comment (pratishta) [reply db/migration/0013_previous_lenny_balinger.sql:2]: My initial hesitation was simply on the principle of not wanting to mess with automatically generated files. I'm unfamiliar with cross referencing the snapshot metadata. I see that we can use something like the `--custom` flag with Drizzle to generate an empty migration file for manual SQL (maybe same as what Tim linked, can't see it rn). I'm fine either way but would be good to agree on as a team for how we wanna deal

**Comment by pratishta** [comment]
Updated to have two migration files, one custom to set incorrect point data to null and another generated from updating the community district geom type schema

**Review by TylerMatteo — DISMISSED** [review]
Lgtm but maybe hold off on merging until an approval from @TangoYankee just to make sure we're all on the same page.

File: db/migration/0013_previous_lenny_balinger.sql
Comment (TangoYankee) [reply db/migration/0013_previous_lenny_balinger.sql]: @pratishta, I'm not sure whether you saw in chat. But, we were talking about actually dropping columns as we're doing two migrations anyway.

**Review by TangoYankee — CHANGES_REQUESTED** [review]
Changing to two migrations

File: db/migration/0013_previous_lenny_balinger.sql
Comment (pratishta) [reply db/migration/0013_previous_lenny_balinger.sql]: Yup

Comment (TangoYankee) [reply db/migration/0013_previous_lenny_balinger.sql]: Did you have questions or concerns? It appears we're still "updating+altering" instead of "dropping+adding"

**Comment by pratishta** [comment]
Updated to have two auto-generated migrations. First from deleting the li_ft column from the schema and generating. Second from adding the li_ft column with the polygon geometry type and generating again.

**Review by TangoYankee — APPROVED** [review]

**Pull request merged**
